app/scripts/constants.js

```diff
@@ -189,6 +189,7 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
     BuildConfigInstantiateFailed: true,
     // Deployment
     DeploymentCancelled: true,
```

Comment:
Does this reason exist if RolloutCancelled is for cancelled deployments

Reply:
My doc indicates that it did in my [first review](https://docs.google.com/document/d/11SYwEtnq0S6PT9FPXZuyktCdVsdoYfU_9lDghkTf02A/edit) of all the events, but I don't see a `c.recorder.Eventf` with `grep -ri 'DeploymentCancellationFailed' pkg` so I'm pretty sure its not real.   I'll remove this.

Reply:
Might have changed from 3.6 -> 3.7